docs/events.md

```diff
@@ -1,42 +1,58 @@
 # Events
 
-The framework provides a number of events to help move along your games progess
+The framework provides a number of events to help move along your games progess.
 
-Some come by default, and can be turned off if you dont want them, others need to
-be explicity turned on to be available
+Some come by default, and can be turned off if you dont want them.  
+Others need to be explicity turned on in order to be available.
 
-when provided, you can call them during gameplay,
-or inside of the config object you pass to `Game`
+When provided by the framework, you can call them during gameplay,
+or inside of the config object you pass to `Game`.
 
-but in both cases how you access them is slightly different
+Although in both cases, how you access them is slightly different.
 
 ### Provided Events
 
-* endTurn(G, ctx) - provided by default, to disable add `endTurn: false` to a custom flow config
-* endPhase(G, ctx) - provided by default when using phases, to disable add `endPhase: false` to a custom flow config
-* endGame(G, ctx) - only provided when `endGame: true` in a custom flow config
-* undo(G, ctx) - provided when given a list of `undoableMoves` in flow config of game
-* redo(G, ctx) - provided when given a list of `undoableMoves` in flow config of game
-* changeActionPlayers(G, ctx) - provided by default when more than one player available to change players who can currently make a move
-
-### triggering events during gameplay
-
-to trigger game events from inside of your game, your Board component is passed an object of event functions as a prop called events
-
-to use just call the event you want to trigger
-
-```
-this.props.events.endTurn()
+* endTurn() - Provided by default, to disable add `endTurn: false` to a custom flow config.
+* endPhase(nextPhase) - Provided by default when using phases,  
+  to disable add `endPhase: false` to a custom flow config. \* nextPhase - optional phase to transition to, if not provided defaults to next phase defined in config
+* endGame(arg) - This is only provided when setting `endGame: true` in a custom flow config.
+  * arg - This optional argument when provided will be available as ctx.gameOver.
+* undo() - Provided when given a list of `undoableMoves` in flow config of game.
```

Comment:
`undo` and `redo` is actually turned on by default, if `undoableMoves` is an empty array, then `undo` and `redo` is not provided, if it has content it restricts `undo` and `redo` to those moves provided.

Reply:
maybe also refer to the the [undo/redo](http://boardgame.io/#/undo) page for more information.

Reply:
perfect, i guess i was thinking of the old api where if you didnt provide `undo: true` you didnt get the events, that was a big reason i wanted to do this, to clairify weird little parts of the api like that, thanks @sladwig